cluster-http/src/test/scala/akka/cluster/http/management/ClusterHttpManagementRoutesSpec.scala

```diff
@@ -365,6 +365,28 @@ class ClusterHttpManagementRoutesSpec
         system.terminate()
       }
     }
+
+    "respect base path" in {
+      val pathPrefixName = "cluster"
+
+      val address1 = Address("akka", "Main", "hostname.com", 3311)
+      val uniqueAddress1 = UniqueAddress(address1, 1L)
+      val clusterMember1 = Member(uniqueAddress1, Set())
+      val members = SortedSet(clusterMember1)
+
+      val mockedCluster = mock(classOf[Cluster])
+      val mockedClusterReadView = mock(classOf[ClusterReadView])
+      when(mockedCluster.readView).thenReturn(mockedClusterReadView)
+      when(mockedClusterReadView.members).thenReturn(members)
+      doNothing().when(mockedCluster).leave(any[Address])
+
+      Seq("akka://Main@hostname.com:3311", "Main@hostname.com:3311").foreach(address => {
```

Comment:
Much-needed test case, thanks! Might be good to test just one address here since this `foreach` sort of detracts from the main point?

Reply:
Good point. Let me do it.

Reply:
Done